featuretools/tests/computational_backend/test_pandas_backend.py

```diff
@@ -515,11 +515,18 @@ def test_where_clause_empty_dataframe():
     es.entity_from_dataframe(entity_id="child", dataframe=child_df, index="id", time_index="time_index")
     es.add_relationship(ft.Relationship(es["parent"]["id"], es["child"]["parent_id"]))
 
+    # create regular agg
+    count = Count(es["child"]['id'], es["parent"])
+
+    # create a agg with where
     where = ft.Feature(es["child"]["value"]) == 1
-    count = Count(es["child"]['id'], es["parent"], where=where)
+    count_where = Count(es["child"]['id'], es["parent"], where=where)
+
+    # create agg feature that requires multiple arguments
+    trend = Trend([es["child"]['value'], es["child"]['time_index']], es["parent"])
 
     # cutoff time before all rows
-    ft.calculate_feature_matrix(entityset=es, features=[count], cutoff_time=pd.Timestamp("12/31/2017"))
+    ft.calculate_feature_matrix(entityset=es, features=[count, count_where, trend], cutoff_time=pd.Timestamp("12/31/2017"))
 
     # cutoff time after all rows, but where clause filters all rows
-    ft.calculate_feature_matrix(entityset=es, features=[count], cutoff_time=pd.Timestamp("1/4/2018"))
+    ft.calculate_feature_matrix(entityset=es, features=[count, count_where, trend], cutoff_time=pd.Timestamp("1/4/2018"))
```

Comment:
Could we add assertions here to confirm we get expected feature values from these matrices?

Reply:
done